gpjax/kernels/__init__.py

```diff
@@ -14,8 +14,9 @@
 # ==============================================================================
 
 """JaxKern."""
-from .approximations import RFF
 from .base import AbstractKernel, ProductKernel, SumKernel
+
+from .approximations import RFF
```

Comment:
Has to come _after_ the `.base` imports so we don't get forward-reference issues!